website/client/components/tasks/taskModal.vue

```diff
@@ -19,10 +19,11 @@
           label(v-once) {{ $t('notes') }}
           textarea.form-control(:class="[`${cssClass}-modal-input`]", v-model="task.notes", rows="3")
       .task-modal-content
-        .option(v-if="task.type === 'reward'")
-          label(v-once) {{ $t('cost') }}
-          input(type="number", v-model="task.value", required, placeholder="1.0", step="0.01", min="0")
-          .svg-icon.gold(v-html="icons.gold")
+        .option.form-group.row(v-if="task.type === 'reward'")
+          label.col-sm-2.col-form-label(v-once) {{ $t('cost') }}
+          .input-group.col-sm-10
```

Comment:
I'd use `col-sm-9` and add `margin-left: 1em;` so that its aligned with the Tags control

Reply:
I had considered adding that, but I realized that it would add css rules in a somewhat unmaintainable way. I'll make the changes and update the PR.